.github/workflows/test.yml

```diff
@@ -41,8 +41,6 @@ jobs:
       if: runner.os == 'macOS'
       run: |
         make web-build
-        # Explicitly set XRootD to IPv4-only on Mac OS X to avoid a subtle config bug.
-        export PELICAN_XROOTD_IPV4ONLY=true
```

Comment:
I'd assume we remove this because there's no corresponding Pelican config for `XRootD.IPV4Only`, but let's confirm with whoever put this here to make sure it won't break anything. Looking at git blame, @bbockelm seems to add this line.

Reply:
Yep, I confirmed with Brian that this is old and hasn't been doing anything for awhile. Should be safe to remove.